users/utils.py

```diff
@@ -37,3 +42,47 @@ def user_has_change_permission(user, *models):
 
 def user_has_delete_permission(user, *models):
     return _user_has_models_perms("delete", models)(user)
+
+
+def is_customer(user):
+    if user.is_staff or user.is_superuser:
+        return False
+    elif (
+        user.groups.count() == 1
+        and user.groups.first().name == settings.CUSTOMER_GROUP_NAME
+    ):
+        return True
+    else:
+        return False
+
+
+def get_berth_customers_group():
+    return Group.objects.get(name=settings.CUSTOMER_GROUP_NAME)
```

Comment:
Can we somehow cache the result of this? Since the value won't probably change often (or ever)

Reply:
Yes, the thought crossed my mind too.

Reply:
fixed